### DISCUSSION TITLE
LunaNode VPS discussion

### DISCUSSION THREAD

**Discussion by poroburu** [Ysurac/openmptcprouter#2604] [Q&A]
> [Lunanode](https://www.lunanode.com/) transfer limit from 1TB to 10TB, speed limited to 100Mb/s (CA, FR)
> https://github.com/Ysurac/openmptcprouter/wiki/Install-or-update-the-VPS

After installing the VPS on a c.1 Debian 11 VM, I get this email:

**Luna Node VM Abuse Alert**

> We detected high outgoing SYN traffic to port 22 from your VM with IP address xxx.xxx.xxx.xxx, indicating that your VM may be infected with malware that is participating in distributed bruteforce SSH attack. As a result, your VM has been shutoff.
> 
> You will need to remove the malware from your VM before the VM can be brought online (otherwise, it will be automatically shutoff again); we recommend completely reinstalling the VM. You may boot your VM without network access to remove the malware or recover files -- to do so, go to the Security Groups tab of the VM and remove the 'default' security group, and then hit Start to boot the VM. You will need to add the 'default' security group back to regain external network connectivity.
> 
> Regards,
> Luna Node

Sharing my experience with openmptcrouter on Lunanode. I could not find any discussion on this topic.

**Comment by Ysurac** [comment]
OpenMPTCProuter doesn't use port 22, but you may have a computer on your network that output to port 22 (malware, nmap,...) and it's seen as VPS output.
